src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -193,27 +216,27 @@ object WdlExpression {
           case "LogicalOr" => s"$lhs || $rhs"
           case "LogicalAnd" => s"$lhs && $rhs"
         }
-      }
-      case a:Ast if a.getName == "FunctionCall" => {
-        val name = a.getAttribute("name").asInstanceOf[Terminal].getSourceString
-        val params = a.getAttribute("params").asInstanceOf[AstList].asScala.toVector.map {a => toString(a, highlighter)}
-        s"${highlighter.function(name)}(${params.mkString(", ")})"
-      }
-      case a:Ast if a.getName == "MemberAccess" => {
+      case a:Ast if a.isFunctionCall =>
```

Comment:
Can you add a space after the colon here and on 222, drop the . before the map and add spaces inside the braces?